internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -381,6 +386,32 @@ func (r *ClusterDeploymentsReconciler) updateIfNeeded(ctx context.Context, clust
 	}
 	return update, reply, err
 }
+func (r *ClusterDeploymentsReconciler) updateInstallConfigOverrides(ctx context.Context, clusterDeployment *hivev1.ClusterDeployment,
+	cluster *common.Cluster) (bool, ctrl.Result, error) {
+	// handle InstallConfigOverrides
+	update := false
+	annotations := clusterDeployment.ObjectMeta.GetAnnotations()
+	installConfigOverrides := annotations[InstallConfigOverrides]
+	if cluster.InstallConfigOverrides != installConfigOverrides {
+		cluster.InstallConfigOverrides = installConfigOverrides
+		update = true
+	}
+	var updatedCluster *common.Cluster
+	var err error
+	if update {
+		updatedCluster, err = r.Installer.UpdateClusterInstallConfigInternal(ctx, installer.UpdateClusterInstallConfigParams{
+			ClusterID:           *cluster.ID,
+			InstallConfigParams: cluster.InstallConfigOverrides,
+		})
+		if err != nil {
+			return handleUpdateError(err)
+		}
+		r.Log.Infof("Updated clusterDeployment %s %s", clusterDeployment.Name, clusterDeployment.Namespace)
```

Comment:
I think it's more common to see the namespace first, often separated with a slash. `default/mycd`